engine/execution/state/bootstrap/bootstrap.go

```diff
@@ -94,27 +94,36 @@ func (b *Bootstrapper) IsBootstrapped(db *badger.DB) (flow.StateCommitment, bool
 	return commit, true, nil
 }
 
-func (b *Bootstrapper) BootstrapExecutionDatabase(db *badger.DB, commit flow.StateCommitment, genesis *flow.Header) error {
+func (b *Bootstrapper) BootstrapExecutionDatabase(
+	db *badger.DB,
+	rootSeal *flow.Seal,
```

Comment:
a seal contains all the data needed for bootstrapping. 

We actually don't need the header, because we only need it's ID